src/core/discrete_space.jl

```diff
@@ -354,12 +354,21 @@ function get_node_contents(coords::Tuple, model::ABM{A,<:DiscreteSpace}) where {
 end
 
 """
-    get_node_agents(x, model)
+    get_node_agents(x, model) → agents
+
 Same as `get_node_contents(x, model)` but directly returns the list of agents
 instead of just the list of IDs.
+
+    get_node_agents(x::Array{Any, 1}, model) → agents
+
+Call `get_node_contents(x, model)` by broadcasting a list of nodes or agents.
```

Comment:
Should this be "Call `get_node_contents(x, model)`"?

Reply:
Hm. Technically it calls `get_node_agents` which in turn calls `get_node_contents`. I guess the broadcasting happens with `get_node_agents` so it would make more sense to have that there.